### DISCUSSION TITLE
Motor model in load flow calculation

### DISCUSSION THREAD

**Discussion by lhernandes** [CURENT/andes#492] [Q&A]
Is there an easy way to use and initialize an induction motor model for load flow calculation (i.e. I'm not interested in starting it, but rather starting the simulation with it already in service)? Is there any routine implemented to perform this task? thanks a lot!

**Comment by cuihantao** [comment]
Induction motor model is already included in load flow calculation. The steady-state solution to its differential equations, i.e., 0 = f(x, y), are solved alongside power flow equations.

You can run the test case and see the output.
